packages/namekit-react/src/components/Identity.tsx

```diff
@@ -12,21 +12,46 @@ import {
 } from "@namehash/nameguard";
 import cc from "classcat";
 
-export interface ProfileLinkConfig {
-  getProfileURL: (address: string) => string;
-  getProfileLink: (address: string, children: ReactNode) => JSX.Element;
+export class ProfileLinkGenerator {
+  private baseURL: string;
```

Comment:
I don't think this should have a "baseURL". This assumes too much about how the URLs will be constructed, especially when considering how they should consider both chain and address.